src/isomorphic/classic/element/ReactElementType.js

```diff
@@ -23,7 +23,7 @@ export type ReactElement = {
   $$typeof: any,
   type: any,
   key: any,
-  ref: any,
+  ref: string | (elem: ?HTMLElement) => void,
```

Comment:
(I understand there's a separate command but it's easy to miss when you are learning or trying it for the first time. I didn't know it existed until someone mentioned it on Twitter a month ago.)